packages/salesforcedx-vscode-replay-debugger/package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "view_checkpoints": "Checkpoints",
```

Comment:
@ruthemmanuelle , this is the panel name. We're calling it checkpoints to match what Developer Console calls it: https://help.salesforce.com/articleView?id=code_dev_console_checkpoints_setting.htm&type=5

Reply:
Sounds great.